test/providers/index.test.js

```diff
@@ -14,14 +14,17 @@ describe('CI Providers', () => {
       args: {},
       envs: {
         CI: true,
+        CI_PROJECT_PATH: 'testOrg/testRepo',
```

Comment:
Since we now test separately , is this loop still needed? It seems this array is getting large. What is it's goal currently?

Reply:
Nope, going to kill this file